src/lib/constants/index.js

```diff
@@ -1,113 +1,131 @@
+import { UI } from '@inrupt/lit-generated-vocab-common';
+
 export const PERMISSIONS = {
   APPEND: 'Append',
   READ: 'Read',
   WRITE: 'Write',
   CONTROL: 'Control'
 };
 
-export const ACL_PREFIXES = {
-  acl: 'http://www.w3.org/ns/auth/acl#',
-  foaf: 'http://xmlns.com/foaf/0.1/',
-  n: 'http://www.w3.org/2006/vcard/ns#',
-  rdf: 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
-};
+// export const ACL_PREFIXES = {
+//   [ACL.PREFIX]: ACL.NAMESPACE,
+//   [FOAF.PREFIX]: FOAF.NAMESPACE,
+//   [RDF.PREFIX]: RDF.NAMESPACE,
+//   n: VCARD.NAMESPACE
+// };
```

Comment:
Why are these code blocks commented out, and not removed ?

Reply:
Simply waiting until everything completed, tested and working (just in case I needed to revert any of them temporarily). But yeah, the whole idea is to remove completely of course.